core/server/data/migrations/versions/4.13/03-populate-members-products-events-table.js

```diff
@@ -0,0 +1,33 @@
+const {chunk} = require('lodash');
+const {createTransactionalMigration} = require('../../utils');
+const logging = require('@tryghost/logging');
+const ObjectId = require('bson-objectid').default;
+
+module.exports = createTransactionalMigration(
+    async function up(connection) {
+        logging.info('Adding members_product_events rows for existing members_products relationships');
```

Comment:
Can we add logging to:

1. log and early-return if `memberProductRelationships.length === 0`
2. log with `memberProductRelationships.length` otherwise

Reply:
Yus!! Will do this now!

Reply:
Done